recipe/meta.yaml

```
@@ -35,150 +33,72 @@ requirements:
         - cctools {{ cctools_version }}.*
-        - cctools_{{ cross_platform }} {{ cctools_version }}.*
     test:
-      requires:
-        - clang {{ llvm_version }}.*
       commands:
-        - test -f $PREFIX/bin/{{ macos_machine }}-ld  # [osx]
-        - echo "int main() {}" > lto.c                                              # [osx]
-        - clang -c lto.c -o lto.o -flto                                             # [osx]
-        - $PREFIX/bin/{{ macos_machine }}-ld lto.o -o lto -lSystem -arch x86_64     # [osx and x86_64]
-        - $PREFIX/bin/{{ macos_machine }}-ld lto.o -o lto -lSystem -arch arm64     # [osx and arm64]
-      downstreams:
-        - gfortran_osx-64  # [osx and cross_platform=="osx-64"]
-    about:
-      home: https://github.com/tpoechtrager/cctools-port
-      license: APSL-2.0
-      license_family: Other
-      license_file: cctools/ld64/APPLE_LICENSE
-      summary: Darwin Mach-O cross linker
-
-  - name: ld64
-    version: {{ ld64_version }}
-    build:
-      skip: True   # [cross_platform != target_platform]
-    script: install-ld64-symlinks.sh
-    requirements:
-      host:
-        - {{ pin_subpackage("ld64_" + cross_platform, exact=True) }}
-      run:
-        - {{ pin_subpackage("ld64_" + cross_platform, exact=True) }}
-      run_constrained:
-        - cctools {{ cctools_version }}.*
-        - cctools_{{ cross_platform }} {{ cctools_version }}.*
-    test:  # [osx]
-      commands:  # [osx]
-        - test -f $PREFIX/bin/ld  # [osx]
+        - test -f $PREFIX/bin/ld
     about:
       home: https://github.com/tpoechtrager/cctools-port
       license: APSL-2.0
       license_family: Other
       license_file: cctools/ld64/APPLE_LICENSE
       summary: Darwin Mach-O native linker
+      description: |
+        ld64 is Apple's linker for creating Mach-O executables and libraries.
+        This native version provides:
+        - Creation of Mach-O executables and dynamic libraries
+        - Support for macOS-specific linking features
+        - Generation of debug information compatible with macOS tools
+        - Dynamic library versioning support
 
   - name: cctools
     version: {{ cctools_version }}
-    script: install-cctools-symlinks.sh
+    script: install-cctools.sh
     build:
       skip: True   # [cross_platform != target_platform]
+      missing_dso_whitelist:
+        - "*/libc++.1.dylib"  # [osx]
     requirements:
       host:
-        # - {{ pin_subpackage("ld64", exact=True) }}
-        - {{ pin_subpackage("cctools_" + cross_platform, exact=True) }}
+        - llvm  {{ llvm_version }}
+        - clang {{ llvm_version }}
+        - libcxx-devel {{ llvm_version }}
+        - {{ pin_subpackage("ld64", exact=True) }}
       run:
         - {{ pin_subpackage("ld64", exact=True) }}
-        - {{ pin_subpackage("cctools_" + cross_platform, exact=True) }}
+        - libcxx-devel  {{ llvm_version }}
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET }}  # [osx]
```

```suggestion
        - __osx >={{ MACOSX_DEPLOYMENT_TARGET }}  # [osx]
      run_constrained:
        # clang might pull in the wrong cctools otherwise
        - clang {{ llvm_version }}.*
```
Not sure this is necessary.